AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_bundler_batch_tests.py

```diff
@@ -191,7 +191,7 @@ def test_WindowsAndMac_BundlesAndBundleSettings_EquivalentOutput(self, workspace
         """
         helper = bundler_batch_helper
         seed_list = os.path.join(workspace.paths.engine_root(), "Assets", "Engine", "SeedAssetList.seed")  # Engine seed list
-        asset = r"levels\testdependencieslevel\level.pak"
+        asset = r"levels\testdependencieslevel\testdependencieslevel.prefab"
```

Comment:
Why is this one a prefab when the others are spawnables?

Reply:
They should have all been .spawnable - Updating